app/controllers/api/application_controller.rb

```diff
@@ -1,7 +1,12 @@
 module Api
   class ApplicationController < ::ActionController::API
+    include Pundit
     before_action :authenticate!
 
+    rescue_from Pundit::NotAuthorizedError do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.